src/Controller/MapController.php

```diff
@@ -17,27 +18,55 @@
  */
 class MapController extends AbstractController
 {
+
+    //Affiche les entité "finness" sur la carte en fonction du tri
+    //et toutes les entités si il n'y a pas de tri
+
     /**
      * @Route("/", name="finess_map", methods={"GET"})
      * @param FinessRepository $finessRepository
      * @return Response
      */
-    public function index(FinessRepository $finessRepository): Response
+    public function index(FinessRepository $finessRepository, Request $request, StrainRepository $strainRepo): Response
     {
         $form = $this->createForm(
             MapSelectionType::class,
             null,
```

Comment:
Did you consider having a dataClass behind the form instead of fetching raw data from it ?